docs/guide/writing-migrations.md

```diff
@@ -47,19 +46,19 @@ Example using `methods(self)`:
 module.exports = {
   extend: '@apostrophecms/piece-type',
   async init(self) {
-    self.apos.migration.add('add-product-description', self.addProductDescription);
+    self.apos.migration.add('add-copyright-notice', self.addCopyrightNotice);
   }
   methods(self) {
     return {
-      async addProductDescription() {
+      async addCopyrightNotice() {
         await self.apos.migration.eachDoc({
-          type: 'product'
+          type: 'article'
         }, async (doc) => {
-          if (doc.description === undefined) {
+          if (doc.copyright === undefined) {
             await self.apos.doc.db.updateOne({
               _id: doc._id
             }, {
-              $set: { description: 'No description available' }
+              $set: { copyright: '©2024 ApostropheCMS. All rights reserved.' }
```

Comment:
Still not a frequent use case anymore (because `def` will automatically get propagated), but I do think we need a simple example and it's valid, so let's go with it.